app/addons/documents/changes/components.react.jsx

```diff
@@ -351,6 +360,18 @@ define([
       };
     },
 
+    onClipboardClick: function (target) {
+      var msg = 'The document ID has been copied to your clipboard.';
```

Comment:
in ES 2015 we should use `let` if `const` is not possible because of a rebinding